module-new/src/components/button/button.theme.css

```diff
@@ -12,95 +12,97 @@
 /* STATUSES */
 
 .arm-button:active {
-  filter: brightness(90%);
+  filter: brightness(85%);
 }
 
 .arm-button:disabled {
   filter: none;
   opacity: 0.6;
 }
 
-.arm-button[data-status="positive"] {
+.arm-button[data-status='positive'] {
   background-color: var(--arm-color-positive);
 }
 
-.arm-button[data-status="negative"] {
+.arm-button[data-status='negative'] {
   background-color: var(--arm-color-negative);
 }
 
-.arm-button[data-status="warning"] {
+.arm-button[data-status='warning'] {
```

Comment:
Again it seems your formatter is working against us, these should all be double quotes